doc/api/url.md

```diff
@@ -921,6 +921,51 @@ console.log(url.format(myURL, { fragment: false, unicode: true, auth: false }));
 // Prints 'https://你好你好/?abc'
 ```
 
+## File URL Utility Functions
+
+When working with `file:///` URLs in Node.js (eg when working with ES modules
+which are keyed in the registry by File URL), the utility functions
```

Comment:
I’d s/registry/module registry/ for clarity

Reply:
module map, not registry